### ISSUE TITLE
Support multiple platforms

### ISSUE THREAD

**Issue by Joker9944** [truecharts/containers#35736]
### Is your feature request related to a problem?

I have a small multi node Kubernetes cluster where I use some TrueCharts helm charts. Some of my nodes are amd64 which works fine. Now I started to add in a few Raspberrypies to take some load of the existing nodes. Since the Pis are arm64 none of the images coming for this image mirror repository will work even when multiple platforms are available on upstream image repos.

### Describe the solution you'd like

Mirror platform images available on upstream image repos.

### Describe alternatives you've considered

Overwriting the image defined in the helm chart -> This does not work since most images have customizations to work in the context of the TrueCharts helm chart.
Pivot to a different helm chart -> Workable but I quite like TrueCharts and want to try make it work first.

### I've read and agree with the following

- [X] I've checked all open and closed issues and my request is not there.
- [X] I've checked all open and closed pull requests and my request is not there.

**Comment by PrivatePuffin** [comment]
Not likely to be added within 12 months.
It means we need to check per container and we also dont have a testing backend to ensure arm64 compatibility.

**Comment by Joker9944** [comment]
Thank you for the straight answer.
I got the workaround with overwriting the images in question working in the end.

**Comment by PrivatePuffin** [comment]
> Thank you for the straight answer. I got the workaround with overwriting the images in question working in the end.

Yeah our helm-charts dont depend on using our mirror/containers, as long as the same container source is used.
